setup.py

```diff
@@ -134,7 +134,9 @@ def initialize_options(self) -> None:
 )
 
 airflow_dependencies = [
-    'apache-airflow>=1.10,<2'
+    # Airflow 1.10.0 has an installation failure that bombs the
```

Comment:
This file, one of the 3 largest in the project, increased in size to 380 lines.  The total size of those files is now 3422 lines (target: 1147).  Is this file complex enough to refactor?